src/luarocks/persist.lua

```diff
@@ -147,17 +144,11 @@ write_table = function(out, tbl, level, field_order)
       end
 
       write_value(out, v, level, sub_order)
-      if type(k) == "number" then
```

Comment:
Is that for parsed version tables in `dependencies` table? That could work. Although isn't that table redundant? `repository` table contains dependencies in string format already...